server/src/bots/telegram/botResponse/startResponse.ts

```diff
@@ -2,10 +2,10 @@ import { getHelpProposalInlineKeyboard } from '../services/keyboard';
 import { greetUser } from '../../../messages/userMessage';
 import { CallBackQueryHandlerWithCommandArgument } from '../models';
 import * as TelegramBot from 'node-telegram-bot-api';
+import Storage from '../../telegram/services/storage';
```

Comment:
it's already instance of the class thus should be from small letter, what do you think?